lib/src/configuration.cc

```diff
@@ -517,32 +514,26 @@ static void validateLogDirPath(fs::path logdir_path) {
             throw Configuration::Error { "log directory is not a directory" };
         }
     } else {
-        throw Configuration::Error { "--logdir '" + logdir_path.string() +
-                                     "' doesn't exist" };
+        throw Configuration::Error { "the log directory '" + logdir_path.string()
```

Comment:
Can we change this to say something like, "Cannot write $logfile: No such file or directory $logdir"? The term log directory doesn't really have that much context on its own when it is not exposed.